Docs/source/running_cpp/parameters.rst

```diff
@@ -103,20 +103,13 @@ distribution or the laser field (see below `Particle initialization` and
 
 The parser reads python-style expressions between double quotes, for instance
 ``"a0*x**2 * (1-y*1.e2) * (x>0)"`` is a valid expression where ``a0`` is a
-user-defined constant and ``x`` and ``y`` are variables. The factor
+user-defined constant and ``x`` and ``y`` are variables. The names are case sensitive. The factor
 ``(x>0)`` is `1` where `x>0` and `0` where `x<=0`. It allows the user to
 define functions by intervals. User-defined constants can be used in parsed
-functions only (i.e., ``density_function(x,y,z)`` and ``field_function(x,y,t)``,
-see below). They are specified with:
+functions only (i.e., ``density_function(x,y,z)`` and ``field_function(X,Y,t)``,
+see below).  For example, parameter ``a0`` can be specified with:
```

Comment:
Should we add 

> The following names cannot be used for constants: `x`, `y`, `z`, `X`, `Y`, `t`

here?

Reply:
And are there other rules that would be specified, e.g., constant names cannot include `-`, etc.?
I can commit this changes.

Reply:
Yes, it's a good idea to state that `x`, `y`, `z`, `X`, `Y` and `t` are reserved by WarpX.  Currently the rule for names (variables and constants) are they consist of letters and numbers and the first character must be a letter.  `-` is not allowed because it will create ambiguity.  We could easily allow `_` in the name if it is desired.  Feel free to commit the change.